source/server/http/admin.cc

```diff
@@ -517,39 +517,47 @@ AdminImpl::statsAsJson(const std::map<std::string, uint64_t>& all_stats,
     stats_array.PushBack(stat_obj, allocator);
   }
 
+  Value histograms_obj;
+  histograms_obj.SetObject();
+  Stats::HistogramStatisticsImpl empty_statistics;
+  rapidjson::Value supported_quantile_array(rapidjson::kArrayType);
+  for (double quantile : empty_statistics.supportedQuantiles()) {
```

Comment:
from an API perspective, the supported quantiles can in theory be overridden per-histogram, though the code doesn't support that now.  Do you want to just put a warning check below as you loop through the histograms, that the quantiles match and at least print a warning if they don't?  I think that'll never fire, though, so test-coverage would be annoying.

Reply:
I have added a validation in tests that validates supported quantiles and computed quantiles are of the same size. I think that should be fine for now.